docs/developer-guide/ksqldb-clients/java-client.md

```diff
@@ -691,6 +693,78 @@ System.out.println(description.writeQueries().size() + " queries write to this s
 System.out.println(description.readQueries().size() + " queries read from this stream/table.");
 ``` 
 
+Get metadata about the ksqlDB cluster
+------------------------------------------------------------------
+
+Starting with ksqlDB 0.16.0, the `serverInfo()` method enables client apps to fetch metadata about
+the ksqlDB cluster. The metadata returned from this method includes the version of ksqlDB the server
+is running, the Kafka cluster id and the ksqlDB service id. For more details, see the 
+[API reference](api/io/confluent/ksql/api/client/Client.html#serverInfo()).
+
+### Example Usage ###
+
+Fetch server metadata:
+```java
+ServerInfo serverInfo = client.serverInfo().get();
+System.out.println("The ksqlDB version running on this server is " + serverInfo.getServerVersion());
+System.out.println("The Kafka cluster this server is using is " + serverInfo.getKafkaClusterId());
+System.out.println("The id of this ksqlDB service is " + serverInfo.getKsqlServiceId());
+``` 
+
+Manage, list and describe connectors
+------------------------------------------------------------------
+
+Starting with ksqlDB 0.18.0, the Java client for ksqlDB supports the following connector operations:
+
+- Creating new connectors by using the `createConnector()` method
+- Dropping existing connectors by using the `dropConnector()` method
+- Listing connectors by using the `listConnectors()` method
+- Describing a specific connector by using the `describeConnector()` method
+
+### Example Usage ###
+
+Create a new connector:
+```java
+Map<String, String> connectorProperties = ImmutableMap.of(
+  "connector.class", "io.confluent.connect.jdbc.JdbcSourceConnector",
+  "connection.url", "jdbc:postgresql://localhost:5432/my.db",
+  "mode","bulk",
```

Comment:
nit: inconsistent spacing in this example -- add a space on this line and the ones below?